scripts/qiita

```diff
@@ -328,7 +328,7 @@ def start(port):
 
     http_server = tornado.httpserver.HTTPServer(Application())
     http_server.listen(port)
-    click.echo("Qiita started on port %s" % port)
+    click.echo("Qiita started on port " + str(port))
```

Comment:
Why this change? if port is an integer just use:
`click.echo("Qiita started on port %d" % port)`